test/integration/container/multimaster_test.go

```diff
@@ -376,8 +404,10 @@ func TestMultimasterSetup(t *testing.T) {
 	if shouldRetagPush(t, registryPort) {
 		run(t, filepath.Join(rootDir, "environments/local-docker-registry/retag_push.sh"), "-p", strconv.Itoa(registryPort))
 	}
-	// FIXME: look this value up more dynamically.
-	const capeiImage = "weaveworks/cluster-api-existinginfra-controller:v0.2.4"
+	release, err := getLatestCAPEIRelease()
```

Comment:
Here, though, it seems great! :-)

Reply:
I think we want the version that matches the go.mod file - otherwise there is the potential to create the seed node with one version and the remaining nodes with a different version.

Reply:
That seems interesting. I'm curious in how that could happen. Would `getLatestCAPEIRelease()` be called many times? It seems just to be called once. 🙂